src/components/Collapse.js

```diff
@@ -4,18 +4,17 @@ import { Link } from 'src/components/common'
 import { icon } from 'src/components/icons'
 
 
-const Collapse = ({ title, buttonStyle, children, ...props }) => {
+const Collapse = ({ title, buttonStyle, children, titleFirst, ...props }) => {
   const [isOpened, setIsOpened] = useState(false)
+  const angleIcon = icon(isOpened ? 'angle-down' : 'angle-right', { style: { marginRight: '0.25rem', flexShrink: 0 } })
+  const titleAndIcon = titleFirst ? [div({ display: 'flex', style: { flexGrow: 1 } }, [title]), angleIcon] : [angleIcon, title]
```

Comment:
Remember that, while children **is** an array, it shouldn't **contain** an array, unless that array has keys.
If you inline this where it's used below (which I recommend you do for simplicity), you'll see that you're putting the array you create here _inside_ the children array of the `Link`.